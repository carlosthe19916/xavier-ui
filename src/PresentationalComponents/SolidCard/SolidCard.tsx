import React from 'react';
import { Card, CardBody } from '@patternfly/react-core';

interface Props {
    title: string;
    description: string;
}

interface State {}

export class SolidCard extends React.Component<Props, State> {
    constructor(props: Props) {
        super(props);
    }

    public render() {
        const { title, description } = this.props;

        return (
            <Card className="xa-c-card-solid">
                <CardBody>
                    <h2 className="pf-c-title pf-m-3xl">{title}</h2>
                    <h3 className="pf-c-title pf-m-1xl">{description}</h3>
                </CardBody>
            </Card>
        );
    }
}
