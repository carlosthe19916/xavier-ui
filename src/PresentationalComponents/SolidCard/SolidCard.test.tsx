import React from 'react';
import { shallow } from 'enzyme';
import { SolidCard } from './SolidCard';

describe('SolidCard', () => {
    it('expect to render', () => {
        const wrapper = shallow(<SolidCard title="My Title" description="My description" />);
        expect(wrapper).toMatchSnapshot();
    });
});
