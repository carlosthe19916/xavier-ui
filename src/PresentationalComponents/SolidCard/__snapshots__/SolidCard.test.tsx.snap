// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SolidCard expect to render 1`] = `
<Card
  className="xa-c-card-solid"
>
  <CardBody>
    <h2
      className="pf-c-title pf-m-3xl"
    >
      My Title
    </h2>
    <h3
      className="pf-c-title pf-m-1xl"
    >
      My description
    </h3>
  </CardBody>
</Card>
`;
